client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -131,6 +132,7 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
     nextButton,
     okButton,
     reportSelector,
+    copyContent,
```

Comment:
Adding a new generic child component which houses all of the modal functionality buttons. 

I thought there must be a way to incorporate buttons without adding a new optional component to Modal component, but couldn't see a way other than adding a new child